src/credentials/credentialsCreator.ts

```diff
@@ -0,0 +1,66 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as nls from 'vscode-nls'
+const localize = nls.loadMessageBundle()
+
+import * as AWS from 'aws-sdk'
+import * as vscode from 'vscode'
+
+const ERROR_MESSAGE_USER_CANCELLED = localize(
```

Comment:
Is this a new pattern we want to use going forward? I don't see why this shouldn't just be an inline localize call.

Reply:
Anywhere we have static text (eg: without tokens) it doesn't make sense to re-resolve the text each time. We do this in a few places, but not consistently.